test/unit/lib/censorDeath.js

```diff
@@ -2,242 +2,7 @@
-    dateOfDeath: null,
-    dateOfDeathQualifier: null,
-    ageAtDeath: null,
-    birthplace: null,
-    deathplace: null,
-    sex: null,
-    address: null,
-    occupation: null,
-    retired: null,
-    maidenSurname: null,
-    causeOfDeath: null,
-    certifiedBy: null,
-    relationshipToPartner: null,
-    aliases: []
-  },
-  informant: {
-    forenames: null,
-    surname: null,
-    address: null,
-    qualification: null,
-    signature: null
-  },
-  registrar: {
-    signature: null,
-    designation: null,
-    subdistrict: null,
-    district: null,
-    administrativeArea: null
-  },
-  coroner: {
-    name: null,
-    designation: null,
-    area: null
-  },
-  partner: {
-    name: null,
-    occupation: null,
-    retired: null
-  },
-  father: {
-    name: null,
-    occupation: null
-  },
-  mother: {
-    name: null,
-    occupation: null
-  },
-  status: {
-    blocked: true,
-    correction: null,
-    marginalNote: null,
-    onAuthorityOfRegistrarGeneral: null
-  },
-  previousRegistration: undefined,
-  nextRegistration: undefined
-}
-
-const redactedDataNotBlocked = {
-  id: 1000,
-  date: "2020-09-23",
-  entryNumber: null,
-  inquestDate: null,
-  deceased: {
-    prefix: null,
-    forenames: "Forename1 Forname2",
-    surname: "SURNAME",
-    suffix: null,
-    ageAtDeath: null,
-    birthplace: null,
-    deathplace: null,
-    dateOfBirth: "2020-09-23",
-    dateOfDeath: "2020-09-23",
-    dateOfDeathQualifier: "On or about",
-    sex: "Indeterminate",
-    occupation: null,
-    retired: null,
-    maidenSurname: null,
-    causeOfDeath: null,
-    certifiedBy: null,
-    relationshipToPartner: null,
-    address: "Current Address",
-    aliases: [{
-      type: 'Stage Name',
-      "prefix": "The Artist Formally Known As...",
-      "forenames": "Prince Rogers",
-      "surname": "Prince Rogers Nelson",
-      "suffix": "Unpronounceable Symbol"
-    }, {
-      type: 'Undercover',
-      "prefix": "Agent",
-      "forenames": "Alec",
-      "surname": "Trevelyan",
-      "suffix": "006"
-    }]
-  },
-  informant: {
-    forenames: null,
-    surname: null,
-    address: null,
-    qualification: null,
-    signature: null
-  },
-  registrar: {
-    signature: null,
-    designation: null,
-    subdistrict: "Subdistrict town",
-    district: "District city",
-    administrativeArea: "Adminshire"
-  },
-  coroner: {
-    name: null,
-    designation: null,
-    area: null
-  },
-  partner: {
-    name: null,
-    occupation: null,
-    retired: null
-  },
-  father: {
-    name: null,
-    occupation: null
-  },
-  mother: {
-    name: null,
-    occupation: null
-  },
-  status: {
-    blocked: false,
-    correction: "None",
-    marginalNote: "None",
-    onAuthorityOfRegistrarGeneral: null
-  },
-  previousRegistration: undefined,
-  nextRegistration: undefined
-}
+const { deathData, deathDataBlockedRecord, blockedDeathData, redactedDeathDataNotBlocked } = require('./test_data/censor-record-test-data')
```

Comment:
I wonder whether moving this out into a separate file is actually helpful. If they are only used in this file, maybe they belong here.

Reply:
If they were going in to separate JSON files, that might make sense.

Reply:
@RobinKnipe: Any thoughts on this?

Reply:
I like the idea of test files not being cluttered with 100's of lines of test data, which makes reading the tests easier IMO...but happy to go with consensus.

Reply:
I know what you mean, but some times you need to see that data to understand the test. Also, I'm not sure having one big file full of test data really improves the situation.

Reply:
OK, added test data into individual tests